### ISSUE TITLE
Today earnings shows wrong Total earnings & Total sales for a download

### ISSUE THREAD

**Issue by vishalck** [easydigitaldownloads/easy-digital-downloads#5480]
On http://localhost:8888/edd/wp-admin/edit.php?post_type=download&page=edd-reports&view=downloads&download-id=68&exclude_taxes=0&range=today&year=2017&year_end=2017&m_start=2&m_end=2&day=15&day_end=28, the "Total earnings for period shown" & "Total sales for period shown" are shown incorrect.

Tested this on a live site as well. I imported sample data from the XML file, placed a couple of orders for "One More Sample Product" (totalling $60) download: https://screencast.com/t/ZG0QdZhW . When the filter "Today" is selected, it shows wrong earnings of $1440: https://screencast.com/t/aO5oklq2

**Comment by pippinsplugins** [comment]
@vishalck could you use the "Recount Stats" tool in Downloads > Tools to recount the earnings and see if that resolves it?

**Comment by vishalck** [comment]
@pippinsplugins Didn't know about that tool, thanks. I selected "Recount Earnings and Sales for a download" & choose the appropriate download. My earnings on the "All Downloads" page for that download did change. But when I went in to view the individual earnings for the download & clicked on "Today" filter, it didn't show the right amount. It shows the same incorrect amount as it was shown before recount.
